cli/pkg/kctrl/cmd/package/repository/list.go

```diff
@@ -113,7 +114,13 @@ func NewSourceValue(pkgr v1alpha1.PackageRepository) uitable.Value {
 	if pkgr.Spec.Fetch != nil {
 		switch {
 		case pkgr.Spec.Fetch.ImgpkgBundle != nil:
-			source = "(imgpkg) " + pkgr.Spec.Fetch.ImgpkgBundle.Image
+			tag, ok := pkgr.Annotations[release.RepoVersionAnnKey]
+			if ok {
+				source = fmt.Sprintf("(imgpkg) (%s) %s", tag, pkgr.Spec.Fetch.ImgpkgBundle.Image)
+			} else {
+				source = fmt.Sprintf("(imgpkg) %s", pkgr.Spec.Fetch.ImgpkgBundle.Image)
+			}
```

Comment:
We don't need the `else` block if we don't remove the first line (i.e. `source = "(imgpkg) " + pkgr.Spec.Fetch.ImgpkgBundle.Image`)

Reply:
Adopted